include/ada_c.h

```diff
@@ -87,7 +87,6 @@ void ada_set_hash(ada_url result, const char* input, size_t length);
 // url_aggregator clear methods
 void ada_clear_port(ada_url result);
 void ada_clear_hash(ada_url result);
-void ada_clear_pathname(ada_url result);
```

Comment:
`ada_clear_pathname` can be implemented as `set_pathname("")`. I think.

Reply:
Yes, I added as a commit `comment` but I don't think we need another function for this.